packages/components/pie-divider/src/index.ts

```diff
@@ -24,8 +26,22 @@ export class PieDivider extends LitElement implements DividerProps {
     @validPropertyValues(componentSelector, orientations, defaultProps.orientation)
     public orientation = defaultProps.orientation;
 
+    @property({ type: String })
+    public label = defaultProps.label;
+
+    renderDividerLabel () {
+        if (!this.label) {
+            return nothing;
+        }
+
+        return html`
+            <hr aria-hidden="true"/>
+            <span class="c-divider-label">${this.label}</span>
```

Comment:
I am sure it does, but does this label get correctly announced by screen readers?

Reply:
https://github.com/user-attachments/assets/0747211e-f7a6-46a1-9c4b-1f0c6913414a